### PR TITLE
fix: Do not add management conflict annotaiton for direct resources

### PR DESCRIPTION
**Pull request by jasonvigil**

### CODE REVIEW

**Comment by yuwenma** [comment]
/approve

Thank you!

File: pkg/gvks/supportedgvks/supportedgvks.go [hunk 1]
```diff
@@ -144,3 +144,17 @@ func BasedOnDCL(serviceMetaLoader metadata.ServiceMetadataLoader) []schema.Group
 	}
 	return gvkList
 }
+
+func IsDirectByGVK(gvk schema.GroupVersionKind) bool {
```
Comment (yuwenma) [inline pkg/gvks/supportedgvks/supportedgvks.go:148]: It's better to pass in gk here

**Review by yuwenma — COMMENTED** [review]
/hold cancel

File: pkg/gvks/supportedgvks/supportedgvks.go (hunk 1 above)
Comment (jasonvigil) [reply pkg/gvks/supportedgvks/supportedgvks.go:148]: Can you please explain why? I thought GVK is more precise.

**Pull request merged**